src/drivers/optical_flow/thoneflow/module.yaml

```diff
@@ -0,0 +1,6 @@
+module_name: ThoneFlow-3901U optical flow sensor
+serial_config:
+    - command: thoneflow start -d ${SERIAL_DEV}
+      port_config_param:
+        name: SENS_TFLOW_CFG
```

Comment:
The parameter setting is not like other serial device have a serial list in QGC, is that need to wait parameter metadata update on QGC side?